pkgs/applications/virtualization/distrobox/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
```

Comment:
I think we can use `stdenvNoCC` here :thinking:

Reply:
Oh I didn't know it existed, thx I'll try!